test/parallel/test-fs-read-stream-file-handle.js

```diff
@@ -80,3 +80,22 @@ fs.promises.open(file, 'r').then((handle) => {
     assert.strictEqual(output, input);
   }));
 }).then(common.mustCall());
+
+fs.promises.open(file, 'r').then((handle) => {
```

Comment:
Nit: would really like us to start including more documentation comments in these tests that identify exactly what assertions are being tested. 

```suggestion
// Verify that an AbortSignal can be passed to `handle.createReadStream`
// and is passed correctly.
fs.promises.open(file, 'r').then((handle) => {
```